### DISCUSSION TITLE
Error when using Firebase Analytics plugin - Cannot find symbol

### DISCUSSION THREAD

**Discussion by devry12** [firebase/flutterfire#11622] [Q&A]
I'm encountering an issue while using the Firebase Analytics plugin version 10.5.0 in my Flutter project. I'm getting the following error:

/Users/user/.pub-cache/hosted/pub.dev/firebase_analytics-10.5.0/android/src/main/java/io/flutter/plugins/firebase/analytics/FlutterFirebaseAnalyticsPlugin.java:166: error: cannot find symbol
            taskCompletionSource.setResult(Tasks.await(analytics.getSessionId()));
            
            
This issue arises after updating the Firebase Analytics plugin to version 10.5.0. It appears that the error is related to the usage of getSessionId(), which seems to no longer be available in the latest version of the plugin.

I'd like to note that my application is running smoothly on the iOS platform, and this issue seems to be specific to the Android platform.

Steps Taken So Far:

I've ensured that I've updated the Firebase Analytics plugin to the latest version in my pubspec.yaml.
I've consulted the official documentation and changelog for the Firebase Analytics plugin, but I'm still unsure about how to resolve this issue.

Questions:

- Are there any changes in how to access the session ID or the method used to retrieve the session ID in the latest version of the Firebase Analytics plugin for the Android platform?
- Could you provide guidance or an example on how to properly address this error in version 10.5.0?

Thank you for your assistance in resolving this issue. Your help and guidance are greatly appreciated.

**Comment by ducgao** [comment]
same here
